cluster/get-kube.sh

```diff
@@ -64,6 +64,34 @@ set -o nounset
 set -o pipefail
 
 KUBERNETES_RELEASE_URL="${KUBERNETES_RELEASE_URL:-https://storage.googleapis.com/kubernetes-release/release}"
+KUBE_DEV_RELEASE_BUCKET_URL="${KUBE_DEV_RELEASE_BUCKET_URL:-https://storage.googleapis.com/kubernetes-release-dev}"
+
+KUBE_RELEASE_VERSION_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(-(beta|alpha)\\.(0|[1-9][0-9]*))?$"
+KUBE_CI_VERSION_REGEX="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)-(beta|alpha)\\.(0|[1-9][0-9]*)(\\.(0|[1-9][0-9]*)\\+[-0-9a-z]*)?$"
+
+# Sets KUBE_VERSION variable to the proper version number (e.g. "v1.0.6",
+# "v1.2.0-alpha.1.881+376438b69c7612") or a version' publication of the form
```

Comment:
The second part of this sentence doesn't make sense.

Basically, this function either sets `KUBE_VERSION` if an explicit version number was provided, or it resolves the "published" version by reading from GCS.

I see that you just copied this comment from `hack/get-build.sh`, but we should probably improve it.